spec/features/see_reviews_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "see reviews", %{
+  As a user,
+  I want to view the reviews for a beer
+  So that my mind can be influenced
+} do
+  # Acceptance Criteria
+  # * I must be on the details page for a specific beer
+  # * I must only see the reviews for the beer I’m viewing
+  # * I must see the reviews listed in order, most recent first
+
+  scenario "sucessfully see reviews" do
+    reviews = Factorygirl.create_list(:review, 2)
+    unrelated_review = Factorygirl.create(:review)
+
+    visit root_path
+    click_link reviews.first.beer
+
+    expect(page).to have_selector(
```

Comment:
Check this out: http://stackoverflow.com/questions/8423576/is-it-possible-to-test-the-order-of-elements-via-rspec-capybara/14769902#14769902

`page.body.index` will return the index of a series of elements. You can see if one element comes before another. This is a bit clearer than doing it by selectors.